src/components/list/examples/list-primary-component.tsx

```diff
@@ -0,0 +1,90 @@
+import { ListItem } from '@limetech/lime-elements';
+import { Component, h } from '@stencil/core';
+
+/**
+ * List with a primary component
+ */
+@Component({
+    tag: 'limel-example-list-primary-component',
+    shadow: true,
+})
+export class ListCircularProgressExample {
+    private actionItems: Array<ListItem<number>> = [
+        { text: 'Go to my fab object', value: 10 },
+        { text: 'Delete object', value: 11 },
+    ];
+
+    private items: Array<ListItem<number>> = [
+        {
+            text: 'King of Tokyo',
+            secondaryText: '2 players',
+            value: 1,
+            actions: this.actionItems,
+            primaryComponent: {
+                name: 'limel-circular-progress',
+                props: {
+                    value: 5,
+                    maxValue: 10,
+                    suffix: '%',
+                    displayPercentageColors: true,
+                },
+            },
+        },
+        {
+            text: 'Smash Up!',
+            secondaryText: '2-5 players',
+            value: 2,
+            primaryComponent: {
+                name: 'limel-circular-progress',
+                props: {
+                    value: 1,
+                    maxValue: 10,
+                    suffix: '%',
+                    displayPercentageColors: true,
+                },
+            },
+        },
+        {
+            text: 'Pandemic',
+            secondaryText: '2-4 players',
+            value: 3,
+            primaryComponent: {
+                name: 'limel-circular-progress',
+                props: {
+                    value: 8,
+                    maxValue: 10,
+                    suffix: '%',
+                    displayPercentageColors: true,
+                },
+            },
+        },
+        {
+            text: 'Ticket to Ride',
+            secondaryText: '1-3 players',
+            value: 5,
+            primaryComponent: {
+                name: 'limel-circular-progress',
+                props: {
+                    value: 3,
+                    maxValue: 10,
+                    suffix: '%',
+                    displayPercentageColors: true,
+                },
+            },
+        },
+    ];
+
+    public render() {
+        return (
+            <limel-list
+                items={this.items}
+                onSelect={this.onSelectAction}
+                class="has-striped-rows"
+            />
+        );
+    }
+
+    private onSelectAction(event: CustomEvent<ListItem>) {
+        console.log('Executing action: ', event.detail);
+    }
+}
```

Comment:
```suggestion
            <limel-list
                items={this.items}
                class="has-striped-rows"
            />
        );
    }
}
```

`fixup! docs(list): create a list example with a custom component`